src/LinuxTracing/UprobesUnwindingVisitor.h

```diff
@@ -81,6 +81,10 @@ class UprobesUnwindingVisitor : public PerfEventVisitor {
     samples_in_uretprobes_counter_ = samples_in_uretprobes_counter;
   }
 
+  void SetInitialTidToRootNamespaceTidMapping(absl::flat_hash_map<pid_t, pid_t>&& tid_mappings) {
+    tid_to_root_namespace_tid_ = tid_mappings;
```

Comment:
```suggestion
    tid_to_root_namespace_tid_ = std::move(tid_mappings);
```
`tid_mappings` wants to be passed an r-value but then in the function it has a name and it's an l-value.

Reply:
Thanks, good catch.